internal/scm/bitbucketcloud/bitbucket_cloud.go

```diff
@@ -78,8 +87,34 @@ func (bbc *BitbucketCloud) CreatePullRequest(ctx context.Context, repo scm.Repos
 }
 
 func (bbc *BitbucketCloud) UpdatePullRequest(ctx context.Context, repo scm.Repository, pullReq scm.PullRequest, updatedPR scm.NewPullRequest) (scm.PullRequest, error) {
-	//TODO implement me
```

Comment:
I think this comment and panic have to go?

Reply:
It's removed on my side, is the diff showing up as included?

Reply:
Ah, on refresh I can see it, something was with my UI it seems like (it was showing as unchanged still), your good